core/src/test/scala/org/apache/spark/deploy/worker/DriverRunnerTest.scala

```diff
@@ -57,11 +56,9 @@ class DriverRunnerTest extends SparkFunSuite {
       superviseRetry: Boolean) = {
     val runner = createDriverRunner()
     runner.setSleeper(mock(classOf[Sleeper]))
-    doAnswer(new Answer[Int] {
-      def answer(invocation: InvocationOnMock): Int = {
-        runner.runCommandWithRetry(processBuilder, p => (), supervise = superviseRetry)
-      }
-    }).when(runner).prepareAndRunDriver()
+    doAnswer((_: InvocationOnMock) =>
```

Comment:
Nit: In this case would not we prefer to have `{`, like:

```
doAnswer { (_: InvocationOnMock) =>
}
```

Reply:
I can change it. I don't have a strong preference. I default to not using blocks where not necessary, but for non-trivial blocks it's probably clearer to use them anyway